src/docs/components/main/README.md

```diff
@@ -1,6 +1,6 @@
 # Main
 
-This component is a shim of [Patternfly PageSection component](https://v2.patternfly.org/documentation/react/components/page).
+This component is a shim of [Patternfly PageSection component](https://www.patternfly.org/v4/documentation/react/overview/notes/#components).
```

Comment:
This link should be: https://www.patternfly.org/v4/documentation/react/components/page/#page-title, the current link you have here is the release notes

Reply:
😬